src/test/testdata/lsl/scientificNotation.lsl

```diff
@@ -0,0 +1,13 @@
+libsl "1.0.0";
+library simple;
+typealias Int = int32;
+typealias float = float32;
+typealias double = float64;
+automaton A : Int {
+    var d1: double = 4.05E-13;
+    var d2: double = 4.05E13;
+    var d3: double = 4.05E+13;
+    var d4: double = -1.5E+2;
+    var d5: double = -2.1E-5;
+    var d6: float = 3.4e+38f;
+}
```

Comment:
I suggest also adding `0.0e+0` just to be extra sure.

Reply:
Fixed;